composer.json

```diff
@@ -158,14 +155,11 @@
         "installer-types": ["bower-asset", "npm-asset"],
         "patches": {
             "drupal/core": {
-                "https://www.drupal.org/project/drupal/issues/2466553": "https://www.drupal.org/files/issues/2019-05-21/2466553-translation_interface-60.patch",
```

Comment:
There 4 patches need to be checked. Are they needed, which version of the patch works for 8.9. These versions of the patches don't work.